scripts/scene/RoomBuilder.gd

```diff
@@ -92,6 +88,9 @@ func setting_door_motion(event: InputEventMouseMotion, offset: Vector2, zoom: Ve
 
 # -- Selection and drawing functions
 
+#func set_roomtype() -> void:
```

Comment:
We're not using this but I'm leaving it as a reminder that we might want to remove the roomtype setting from `selecting_tile` and put it in its own function...